mapboxgl/templates/map.html

```diff
@@ -1,29 +1,302 @@
-{% extends "base.html" %}
-
-{% block legend %}{% endblock legend %}
-
-{% block map %}
-
-    map.on('style.load', function() {
-        
-        // Add data source
-        map.addSource("data", {
-            "type": "geojson",
-            "data": {{ geojson_data }},
-            "buffer": 1,
-            "maxzoom": 14
-        });
-        
-        // Add data layer
-        map.addLayer({
-            "id": "circle",
-            "source": "data",
-            "type": "circle",
-            "maxzoom": {{ maxzoom }},
-            "minzoom": {{ minzoom }},
-            "paint": { "circle-radius": 1 }
-        });
+<!DOCTYPE html>
+<html>
+<head>
+<title>mapboxgl-jupyter viz</title>
+<meta charset='UTF-8' />
+<meta name='viewport'
+      content='initial-scale=1,maximum-scale=1,user-scalable=no' />
+<script type='text/javascript'
+        src='https://api.tiles.mapbox.com/mapbox-gl-js/{{ gl_js_version }}/mapbox-gl.js'></script>
+<link type='text/css'
+      href='https://api.tiles.mapbox.com/mapbox-gl-js/{{ gl_js_version }}/mapbox-gl.css' 
+      rel='stylesheet' />
 
+<style type='text/css'>
```

Comment:
probably it makes sense to separate CSS and JS code at least to different `{% include %}` tags

Reply:
Yes, agreed